content/reference/sql/access-management.md

```diff
@@ -0,0 +1,289 @@
+          "GRANT",
+          "REFERENCES",
+          "INDEX",
+          "ALTER",
+          "SHOW DATABASES",
+          "SUPER",
+          "CREATE TEMPORARY TABLES",
+          "LOCK TABLES",
+          "EXECUTE",
+          "REPLICATION SLAVE",
+          "REPLICATION CLIENT",
+          "CREATE VIEW",
+          "SHOW VIEW",
+          "CREATE ROUTINE",
+          "ALTER ROUTINE",
+          "CREATE USER",
+          "EVENT",
+          "TRIGGER",
+          "CREATE TABLESPACE",
+          "CREATE ROLE",
+          "DROP ROLE"
+        ],
+        "Databases": []
+      },
+      "Plugin": "mysql_native_password",
+      "Password": "*A0F874BC7F54EE086FCE60A37CE7887D8B31086B",
+      "PasswordLastChanged": "1970-01-01T00:00:01Z",
+      "Locked": false,
+      "Attributes": null,
+      "IsRole": false
+    },
+    {
+      "User": "test_role",
+      "Host": "%",
+      "PrivilegeSet": {
+        "GlobalStatic": [
+          "SELECT",
+          "INSERT",
+          "UPDATE",
+          "DELETE"
+        ],
+        "Databases": []
+      },
+      "Plugin": "mysql_native_password",
+      "Password": "",
+      "PasswordLastChanged": "2022-02-15T00:00:00.0000000Z",
+      "Locked": true,
+      "Attributes": null,
+      "IsRole": true
+    },
+    {
+      "User": "test_user",
+      "Host": "localhost",
+      "PrivilegeSet": {
+        "GlobalStatic": [],
+        "Databases": [
+          {
+            "Name": "some_db",
+            "Privileges": [],
+            "Tables": [
+              {
+                "Name": "some_tbl",
+                "Privileges": [
+                  "DROP"
+                ],
+                "Columns": []
+              }
+            ]
+          },
+          {
+            "Name": "other_db",
+            "Privileges": [
+              "CREATE",
+              "DROP"
+            ],
+            "Tables": []
+          }
+        ]
+      },
+      "Plugin": "mysql_native_password",
+      "Password": "",
+      "PasswordLastChanged": "2022-02-15T00:00:00.0000000Z",
+      "Locked": false,
+      "Attributes": null,
+      "IsRole": false
+    }
+  ],
+  "Roles": [
+    {
+      "FromHost": "%",
+      "FromUser": "test_role",
+      "ToHost": "localhost",
+      "ToUser": "test_user",
+      "WithAdminOption": false
+    }
+  ]
+}
+```
+
+## Updates and Persistence
+
+Unlike in MySQL, Dolt immediately processes all updates to the grant tables.
+Any privilege changes will immediately take effect, any deleted users with current sessions will immediately lose access, etc.
+Whether these are benefits or drawbacks depend on those running the database.
+For Dolt, the decision to have all updates take immediate effect allows for emergency updates to not require a server restart in some cases, which we believe offers some security and convenience advantages.
+The benefit of delayed updates do not seem as likely nor often, although we may still change this behavior in the future if it is proven otherwise.
```

Comment:
(minor) "nor" always sounds odd to me when not used together with "neither". A couple of searches online showed similar guidance:
https://thewritepractice.com/how-to-use-either-neither-or-and-nor-correctly/